Showcase/Showcase/Pages/Tokens/Color/NamedColor/NamedColor+Chart.swift

```diff
@@ -0,0 +1,205 @@
+                return theme.colorChartSequentialAccent3Tint300
+            case .colorChartSequentialAccent3Tint400:
+                return theme.colorChartSequentialAccent3Tint400
+            case .colorChartSequentialAccent3Tint500:
+                return theme.colorChartSequentialAccent3Tint500
+            case .colorChartSequentialAccent3Tint600:
+                return theme.colorChartSequentialAccent3Tint600
+            case .colorChartSequentialAccent3Tint700:
+                return theme.colorChartSequentialAccent3Tint700
+            case .colorChartSequentialAccent3Tint800:
+                return theme.colorChartSequentialAccent3Tint800
+            case .colorChartSequentialAccent3Tint900:
+                return theme.colorChartSequentialAccent3Tint900
+            case .colorChartSequentialAccent4Tint100:
+                return theme.colorChartSequentialAccent4Tint100
+            case .colorChartSequentialAccent4Tint200:
+                return theme.colorChartSequentialAccent4Tint200
+            case .colorChartSequentialAccent4Tint300:
+                return theme.colorChartSequentialAccent4Tint300
+            case .colorChartSequentialAccent4Tint400:
+                return theme.colorChartSequentialAccent4Tint400
+            case .colorChartSequentialAccent4Tint500:
+                return theme.colorChartSequentialAccent4Tint500
+            case .colorChartSequentialAccent4Tint600:
+                return theme.colorChartSequentialAccent4Tint600
+            case .colorChartSequentialAccent4Tint700:
+                return theme.colorChartSequentialAccent4Tint700
+            case .colorChartSequentialAccent4Tint800:
+                return theme.colorChartSequentialAccent4Tint800
+            case .colorChartSequentialAccent4Tint900:
+                return theme.colorChartSequentialAccent4Tint900
+            case .colorChartSequentialAccent5Tint100:
+                return theme.colorChartSequentialAccent5Tint100
+            case .colorChartSequentialAccent5Tint200:
+                return theme.colorChartSequentialAccent5Tint200
+            case .colorChartSequentialAccent5Tint300:
+                return theme.colorChartSequentialAccent5Tint300
+            case .colorChartSequentialAccent5Tint400:
+                return theme.colorChartSequentialAccent5Tint400
+            case .colorChartSequentialAccent5Tint500:
+                return theme.colorChartSequentialAccent5Tint500
+            case .colorChartSequentialAccent5Tint600:
+                return theme.colorChartSequentialAccent5Tint600
+            case .colorChartSequentialAccent5Tint700:
+                return theme.colorChartSequentialAccent5Tint700
+            case .colorChartSequentialAccent5Tint800:
+                return theme.colorChartSequentialAccent5Tint800
+            case .colorChartSequentialAccent5Tint900:
+                return theme.colorChartSequentialAccent5Tint900
+            }
+        }
+    }
+}
```

Comment:
Normalement tu dois avoir des warnings tels que :
- "Blanket Disable Command Violation: The disabled 'function_body_length' rule should be re-enabled before the end of the file (blanket_disable_command)"

Les warnings désactivés dans un fichier doivent être réactivés ensuite.

Il faut ajouter à la fin du fichier, après un saut de ligne, les lignes : 

```swift
// swiftlint:enable function_body_length
```